app/translations/en.json

```diff
@@ -1,6 +1,6 @@
 {
   "zingo": "Zingo!",
-  "version": "zingo-1.3.3 (131)",
+  "version": "zingo-1.3.4 (137)",
```

Comment:
Is 137 the right number?

Reply:
yeap, you can see it in your Testflight info in your iPhone.